### PR TITLE
Better/Simple(/Stronger) wombat setup.

### PR DESCRIPTION
**Pull request by mgautierfr**
No real functional change. Only code polishing.

Edit: Important code polishing, we now have removed the duplicated definition of fuzzy rules.

### CODE REVIEW

**Comment by kelson42** [comment]
@mgautierfr Why the fuzzy rules, both the one for Python and the one in javascript, are not data-driven? That would allow:
* to avoid sync errors between the two world
* easy update
* easy/proper tracking of the rules

File: src/warc2zim/statics/wombat_setup.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    for (const rule of fuzzy_rules) {
+      const new_path = path.replace(rule.match, rule.replace);
+      if (new_path != path) {
+        return new_path;
+      }
+    }
+    return path;
+  };
+
+  const getWombatInfo = function (
```
Comment (benoit74) [inline src/warc2zim/statics/wombat_setup.js:63]: Having a variable named `getWombatInfo` inside another variable named `getWombatInfo` does not really help in readability, it took me a while to understand this relation.

File: src/warc2zim/statics/wombat_setup.js [hunk 2]
```diff
@@ -0,0 +1,197 @@
+// This file setup a wbinfo dictionnary to configure a wombat instance.
+// It mainly export one function `getWombatInfo` which takes context information
+// and returns a configuration dictionnary to pass to wombat.
+
+const getWombatInfo = (function () {
+  // Reduce "complex" url to our simplyfied ones.
+  // ! MUST be in sync with the python code (at zim creation time)
+  let reduce = function (path) {
```
Comment (benoit74) [inline src/warc2zim/statics/wombat_setup.js:8]: I don't get why this is not a `const`?

**Review by benoit74 — CHANGES_REQUESTED** [review]
Some minor changes + I second the question of @kelson42 about why the wombat_setup.js is not a template solved by Jinja. so that we can declare all fuzzyrules in a more "data-driven" and unique place.

I did not added again all remarks made in previous PR by renaud ^^

File: src/warc2zim/statics/wombat_setup.js (hunk 1 above)
Comment (mgautierfr) [reply src/warc2zim/statics/wombat_setup.js]: This is now replaced by a "direct" return:

```js
return function(
...
```

File: src/warc2zim/statics/wombat_setup.js (hunk 2 above)
Comment (mgautierfr) [reply src/warc2zim/statics/wombat_setup.js]: Keyboard mistake. I swear (please trust me), I've typed `const` but the keyboard writes `let` instead.

This is fixed.

**Comment by mgautierfr** [comment]
> Why the fuzzy rules, both the one for Python and the one in javascript, are not data-driven? 

This is now the case with the last commit.

**Review by benoit74 — APPROVED** [review]
LGTM, thank you

**Pull request merged**